web/src/main/java/com/google/sps/servlets/CommentsUrl.java

```diff
@@ -0,0 +1,64 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.FullEntity;
+import com.google.cloud.datastore.KeyFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+// Get the urls of the latest deiscussion sections from reddit wallstreet bets
+@WebServlet("/store-comments-urls")
+public class CommentsUrl extends HttpServlet {
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+    KeyFactory keyFactory = datastore.newKeyFactory().setKind("Comments-url");
+
+    String Reddit =
+        "https://www.reddit.com/r/wallstreetbets/search?q=flair_name%3A%22Daily%20Discussion%22&restrict_sr=1&sort=new";
+    Document redditPage = Jsoup.connect(Reddit).get();
+    System.out.printf("Title: %s\n", redditPage.title());
+
+    // Get the links for the daily discussion
+    Elements dailyDiscussion = redditPage.getElementsByClass("_1poyrkZ7g36PawDueRza-J");
+    Elements links = dailyDiscussion.select("a[href]");
+
+    int urlCount = 0;
+    int urlCurrIndex = 0;
+    for (Element link : links) {
+      String url = link.attr("href");
+
+      // Every three url's we have a url for the daily discussions.
+      if (urlCount == 2 || urlCount == urlCurrIndex + 3) {
```

Comment:
No action required: I think this is a brittle way to parse the page but it might be okay for now.